src/css/styles.css

```diff
@@ -0,0 +1,68 @@
+/* stylelint-disable indentation */
+/* stylelint-disable csstree/validator */
+
```

Comment:
- [ ] Great job on completing the first step of your leaderboard project as specified in the requirements! 👏 However, I would like to emphasize the importance of keeping `stylelint` enabled. Stylelint plays a crucial role in helping you write industry-standard styles for your project, ensuring consistency and maintainability.

I understand that you mentioned disabling `stylelint` in the attached PR, but I would recommend against doing so. Instead, let's work together to address these errors and improve your code. I'm here to help you learn and understand how to fix them, which will ultimately enhance your coding skills.

To address these errors, you can use the `--fix` flag, which automatically corrects simple style errors. For stylelint-specific issues, you can utilize the following command:

```bash
npx stylelint "**/*.{css,scss}" --fix
```

By running this command, `stylelint` will fix the indentation issues you have automatically, making your styles error-free. 😊😊

Reply:
Hello @Strangeal 
Thank you so much for your review!

I didn't know how to fix that without disabling the linter rule, thank you very much.